packages/integration-tests/tsconfig.json

```diff
@@ -1,12 +1,22 @@
 {
   "extends": "../../tsconfig.base.json",
-  "compilerOptions": { "rootDir": "src", "outDir": "lib", "allowJs": true },
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "lib",
+    "allowJs": true,
+    "paths": {
+      "@env/*": [
+        "./src/test-projects/data-storage-auth-with-triggers-ts/.amplify/function-env/*"
+      ]
+    }
```

Comment:
This makes Typescript happy enough to let us build `integration-tests` with the type definition file added to `data-storage-auth-with-triggers-ts`

Reply:
I was trying to initially but not good enough with tsconfig files to get this to work. I'll experiment some more to get this working.